frontend/stories/PatchPage.stories.js

```diff
@@ -0,0 +1,163 @@
+      url: 'https://via.placeholder.com/150'
+    }
+  },
+  {
+    image: {
+      url: 'https://via.placeholder.com/150'
+    }
+  },
+  {
+    image: {
+      url: 'https://via.placeholder.com/150'
+    }
+  },
+  {
+    image: {
+      url: 'https://via.placeholder.com/150'
+    }
+  },
+  {
+    image: {
+      url: 'https://via.placeholder.com/150'
+    }
+  }
+];
+const teamData = {
+  side_image: {
+    url:
+      'https://res.cloudinary.com/malngaawy/image/upload/v1595261682/Squiggles_Monochromatic_jwvct9.png'
+  },
+  title: 'Team',
+  paragraph:
+    'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Molestie aliquam iaculis facilisis sit pharetra pellentesque ullamcorper.',
+  team_members: [
+    {
+      id: '1',
+      image:
+        'https://images.unsplash.com/photo-1438761681033-6461ffad8d80?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&w=1000&q=80',
+      title: 'Yara',
+      subtitle: 'Front End Developer',
+      description: 'A job listings and job application app for working abroad.'
+    },
+    {
+      id: '2',
+      image:
+        'https://images.unsplash.com/photo-1438761681033-6461ffad8d80?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&w=1000&q=80',
+      title: 'Yara',
+      subtitle: 'Front End Developer',
+      description: 'A job listings and job application app for working abroad.'
+    },
+    {
+      id: '3',
+      image:
+        'https://images.unsplash.com/photo-1438761681033-6461ffad8d80?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&w=1000&q=80',
+      title: 'Yara',
+      subtitle: 'Front End Developer',
+      description: 'A job listings and job application app for working abroad.'
+    }
+  ]
+};
+const joinNextPatchData = {
+  title: 'Join Next Patch',
+  description: 'A job listings and job application app for working abroad.',
+  mentorBtn: 'Apply as Mentor',
+  memberBtn: 'Apply as Member',
+  left_img:
+    'https://s3-alpha-sig.figma.com/img/6ab3/7f82/5cedbb0153583a74429f439e480c8d2a?Expires=1596412800&Signature=OQHfF1fV7IspMfGtbXN6n0d6xjZJ~NSW8SJMsApIvve2nwawX9~2GVO3PkuSmrSFKJyoO958QrJlHKNdPl9wEnydQ2RNtu8IyRtsQNQ0hNesuLMUUF8m7KK42mIorUMMpSsCLU7GEK671lhAl1R4Iliyk~GBmMb197JXNCK4nspcieahLV5xnI-285IpU5mFx2JiOOjqKlcDkVKCGqOAvO0KAN80tywMwcEOisB5E7RqzZ0MQiAAi1gX1lXq~MfBLjaq1KpvKS~2XlxDTyuTgTmEIkwYZloeBIhVwgGULGVyzYfQy5IUzH5YFJ9-Nd-GIN5FWtlfsYBKnhz0eq6k0g__&Key-Pair-Id=APKAINTVSUGEWH5XD5UA',
+  right_img:
+    'https://s3-alpha-sig.figma.com/img/a1ba/aaf1/75daa654aef922b1f1ea60ae6d402855?Expires=1596412800&Signature=hlXZ9MIzIg0wynLj~xmoD4wSnquIIxJDhcfsVLG97RJXZpCSb1TAMi5bgkAG3JcVbylbkNcCHySe13XwVJCqsWMmIIlM8IynoanL3JBhujTtwOCefuiKpCk4I2fE57HgWSVDFatu0WWmarXwSPgjf6-VQiz5YFIF3oGylX1bSeY7bmZjcKP4dBCOgAwFRSjD2zjlzfyjM~N0H6nNfhiLUq-poUB~bbCgX9VUoTwDs446iB6Geq8kfohohOCvm-1hF~NSh2EUf9ENKWAOHCs7oVg6VvsZT3Kf9LJA8schQXwWL2JaYAs-krwFonkbCNAzJHtz-KZI0BsjrXQeftGHsw__&Key-Pair-Id=APKAINTVSUGEWH5XD5UA'
+};
+export const PatchPageComponent = () => {
+  return (
+    <>
+      <div className="container">
```

Comment:
if we will use the main `container` on all page, not for every component 
you have to remove the TopBar container and any component's container else

Reply:
you can't remove container from TopBar because in one of its variations it has a colored background...I suggest adding container to a div wrapping every section that needs it

Reply:
Okay, do it __ remove the main container and add it to the needed sections .